cpu/samd21/cpu.c

```diff
@@ -22,6 +22,19 @@
 #include "periph_conf.h"
 #include "periph/init.h"
 
+#if CLOCK_USE_PLL
+#define CLOCK_8MHZ          1
+#define GEN2_ULP32K         1
+#endif
```

Comment:
Couldn't this block be removed? `CLOCK_8MHZ` and `GEN2_ULP32K` get defined anyways in the following.